### DISCUSSION TITLE
Orion with composite Primary key?

### DISCUSSION THREAD

**Discussion by alekals** [tailflow/laravel-orion#196] [Help]
hello, I have very happily used orion with eloquent and swagger for an API writing test project and I fell in love with this project.
Unfortunately now I find myself using for some projects a database that uses compound primary calls.
So can I use orion with DOCTRINE?
Otherwise can you recommend me a project that does what orion does with support for composite primary key relationships?
Thanks Alex

**Comment by alexzarbn** [comment]
Hi @alekals,

No, currently Orion relies solely on Eloquent and is built on top of it.
I haven't found a package with the same level of automation as Orion, that's why I built it, so can't really recommend any :)
